cmd/bchctl/config.go

```diff
@@ -68,9 +68,9 @@ func listCommands() {
 			continue
 		}
 
-		// Categorize the command based on the usage flags.
+		// Categorize the command based on the usage usageFlags.
```

Comment:
Also i'd just update this comment to say `// Categorize the command based on the usageFlags.`